README.md

```diff
@@ -50,10 +50,10 @@ passed to the [loader](#loader) constructor. In the case of identical property
 names, values from the *options* argument for this method will be used.
 
 ```js
-var loader = vega.loader();
-loader.load('data.json').then(function(data) {
+const loader = vega.loader();
```

Comment:
We usually use ES5 in examples to ensure compatibility with old browsers.

Reply:
Okay. I have updated the commit.